yarn.lock

```diff
@@ -8590,14 +8585,6 @@ jsonschema@^1.4.0:
     array-includes "^3.1.5"
     object.assign "^4.1.3"
 
-kill-port@^2.0.1:
```

Comment:
Was in root deps. We don't depend on kill-port ATM:

https://github.com/blockprotocol/blockprotocol/blob/7dff54059730cdf507df669a2e0eafcba4c8e91c/.github/workflows/ci.yml#L266-L268

Context: https://github.com/blockprotocol/blockprotocol/pull/464